app/scripts/controllers/eventRegistrations.js

```diff
@@ -386,7 +386,15 @@ angular.module('confRegistrationWebApp')
       }
 
       //update registration
-      $http.put('registrations/' + registrant.registrationId, $scope.getRegistration(registrant.registrationId)).error(function(){
+      var registrationIndex = _.findIndex($scope.registrations, { 'id': registrant.registrationId });
+      var registrantIndex = _.findIndex($scope.registrations[registrationIndex].registrants, { 'id': registrant.id });
+      $scope.registrations[registrationIndex].registrants[registrantIndex] = registrant;
```

Comment:
What does this and the 2 lines above it do? It seems like this was already accomplished in the old version. Does modifying the registrant not update the registrations object? Is it created using pass by value and not reference somewhere? I think JS passes objects by reference but I'm not certain.